src/gui/transferlistdelegate.cpp

```diff
@@ -70,6 +70,11 @@ void TransferListDelegate::paint(QPainter * painter, const QStyleOptionViewItem
     QStyleOptionViewItem opt = QItemDelegate::setOptions(index, option);
     QItemDelegate::drawBackground(painter, opt, index);
     switch (index.column()) {
+    case TorrentModel::TR_NAME:
+    case TorrentModel::TR_SAVE_PATH:
```

Comment:
Although i find it ugly, coding guidelines says:
```
case TorrentModel::TR_SAVE_PATH: {
        opt.textElideMode = Qt::ElideMiddle;
        QItemDelegate::paint(painter, opt, index);
        }
        break;
```
The cases below look better to me.